### PR TITLE
feat(misc): owners claim with body, batch claim and export model

### PR DESCRIPTION
**Pull request by daddykotex**
Refs MM-6997, MM-7420, MM-8185

### CODE REVIEW

**Comment by daddykotex** [comment]
By the way, fixing the test runner at 3.5.0 seems to have fixed the issue

**Comment by daddykotex** [comment]
@dmasse76 if you have a minute

**Comment by daddykotex** [comment]
This is ISO with the Java SDK as far as I know .NET.

**Comment by jtheoof** [comment]
@dmasse76 or @mvonthron what do you think?

File: README.md [hunk 1]
```diff
@@ -102,6 +102,41 @@ You can use the `client.Owners` synchronous methods:
 	//Claim an object
     client.Owners.Claim("theUserName", "theDeviceId");
 
+	//Claim an object while overriding some properties
```
Comment (dmasse76) [inline README.md]: Is that expected to have a tab before comments?

Comment (daddykotex) [inline README.md]: ill fix that

Comment (daddykotex) [inline README.md]: There was a mess between tabs and spaces

**Comment by mvonthron** [comment]
LGTM

**Pull request merged**